docs/mkdocs/docs/features/trailing_commas.md

```diff
@@ -0,0 +1,89 @@
+# Trailing Commas
+
+Like comments, this library does not support trailing commas in arrays and objects *by default*.
+
+You can pass set parameter `allow_trailing_commas` to `#!c true` in the parse function to allow trailing commas in arrays and objects. Trailing commas are ignored during parsing.
```

Comment:
Either "pass" or "set", not both.  `#!cpp` everywhere else.

Reply:
I refer to `comments.md`. If it's wrong here, it's alse wrong there...

Reply:
You are right, it's wrong in both places.